Aspire Budgeting/Views/Dashboard/DashboardCardView.swift

```diff
@@ -1,39 +1,119 @@
 //
-//  DashboardCardVIew.swift
+//  DashboardCardView.swift
 //  Aspire Budgeting
 //
 
 import SwiftUI
 
 struct DashboardCardView: View {
-  struct Totals {
-    var availableTotal: AspireNumber
-    var budgetedTotal: AspireNumber
-    var spentTotals: AspireNumber
-  }
 
-  let categoryName: String
-  let totals: Totals
-  let categoryRows: [Category]
+  let cardViewItem: BaseCardView<DashboardCardView>.CardViewItem
+  let baseColor: Color
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/ab414e6b821895ee43e8c1c80baa6a7893f3ce2e/Aspire Budgeting/Views/Dashboard/DashboardCardView.swift#L12"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>DashboardCardView.swift:12</code></td>
    </tr>
  </tbody>
</table>